src/index.ts

```diff
@@ -47,6 +47,11 @@ async function setEnvVars(): Promise<void> {
   }
   if (core.getBooleanInput('experimental')) set('MISE_EXPERIMENTAL', '1')
 
+  const enable = core.getInput('enable_tools')
```

Comment:
needs tests

Reply:
I didn't find any tests - do you have a good starting point?

Reply:
https://github.com/jdx/mise-action/blob/main/.github/workflows/test.yml